zone.js

```diff
@@ -114,38 +114,25 @@ Zone.patchSetClearFn = function (obj, fnNames) {
   }).
   forEach(function (name) {
     var setName = 'set' + name;
-    var clearName = 'clear' + name;
     var delegate = obj[setName];
 
     if (delegate) {
+      var clearName = 'clear' + name;
       var ids = {};
 
-      if (setName === 'setInterval') {
-        zone[setName] = function (fn) {
-          var id;
-          arguments[0] = function () {
-            delete ids[id];
```

Comment:
... and the expected behavior should be `dequeueTask()` always called (whenever clear is called), right ?